internal/handler/queue.go

```diff
@@ -9,35 +10,83 @@ import (
 
 	"github.com/SimplQ/simplQ-golang/internal/datastore"
 	"github.com/SimplQ/simplQ-golang/internal/models"
+	"github.com/go-chi/chi/v5"
 )
 
 func GetQueue(w http.ResponseWriter, r *http.Request) {
-    fmt.Fprintf(w, "GET Queue not implemented")
+	id := r.Context().Value("id").(string)
+	if id == "" {
+		w.WriteHeader(http.StatusNotFound)
+		return
+	}
+	q := datastore.Store.ReadQueue(models.QueueId(id))
+	json.NewEncoder(w).Encode(q)
+	fmt.Fprintf(w, "Get queue")
```

Comment:
what does line 24 do?

Reply:
You are right. Line 24 is not needed. I will remove it.

Reply:
Done.